genesis.go

```diff
@@ -0,0 +1,110 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+)
+
+type GenesisAccount struct {

```

Comment:
Same as previous comment re new sdk updates and module changes since we started - Can we leave this fix for a later phase of the project, where we will update to the latest SDK and include all new module/app patterns?

Reply:
issue created - https://github.com/cosmos/peggy/issues/52